src/hostedPages/activities/wizardForm/WizardForm.tsx

```diff
@@ -1,130 +1,7 @@
-
-  /**
-   * For certain questions like Single Select or Boolean questions,
-   * we don't want the user to click on the answer first and then
-   * on the 'next' button. Instead, we just submit the answer when
-   * user clicks on one of the options and move to the next question.
-   *
-   * Except for last question, there user will have to click on submit
-   * button explicitly.
-   */
-  const submitAndMoveToNextQuestion = () => {
-    if (!isLastQuestion) {
-      handleGoToNextQuestion()
-    }
-  }
-
-  useEffect(() => {
-    determineShowScrollHint()
-  }, [currentQuestion])
-
-  return (
-    <>
-      <main
-        id="ahp_main_content_with_scroll_hint"
-        className={layoutClasses.main_content}
-      >
-        <div className={`${classes.awell_wizard_form} ${classes.container}`}>
-          <div className={classes.form_progress}>
-            <ProgressIndicator percentageCompleted={percentageCompleted} />
-          </div>
-          {isEvaluatingQuestionVisibility ||
-          currentQuestion?.id === undefined ? (
-            <div className={classes.loadingContainer}>
-              <CircularSpinner size="sm" />
-            </div>
-          ) : (
-            <div className={classes.wizard_form}>
-              <Question
-                question={currentQuestion}
-                control={control}
-                getValues={getValues}
-                key={currentQuestion.id}
-                errors={errors}
-                labels={questionLabels}
-                questionTypeConfig={questionTypeConfig}
-                submitAndMoveToNextQuestion={submitAndMoveToNextQuestion}
-              />
-            </div>
-          )}
-        </div>
-      </main>
-      <HostedPageFooter showScrollHint={showScrollHint}>
-        {!isEvaluatingQuestionVisibility && (
-          <div className={`${classes.button_wrapper} ${classes.container}`}>
-            <div>
-              {!isFirstQuestion && (
-                <Button
-                  variant="tertiary"
-                  onClick={handleGoToPrevQuestion}
-                  data-cy="navigateToPrevQuestionButton"
-                >
-                  {buttonLabels.prev}
-                </Button>
-              )}
-            </div>
-            {isLastQuestion ? (
-              <Button
-                onClick={submitForm}
-                type="submit"
-                data-cy="submitFormButton"
-              >
-                {buttonLabels.submit}
-              </Button>
-            ) : (
-              <Button
-                variant="secondary"
-                onClick={handleGoToNextQuestion}
-                data-cy="navigateToNextQuestionButton"
-              >
-                {buttonLabels.next}
-              </Button>
-            )}
-          </div>
-        )}
-      </HostedPageFooter>
-    </>
-  )
-}
+/**
+ * @deprecated This component is no longer recommended for use.
+ * Please use ConversationalForm directly instead.
```

Comment:
It would help to indicate why this was deprecated (e.g. as we now support long-form forms) and that the recommended ConversationalForm works in the same way (i.e. can they just change the import and component name, or has the API changed?)